guides/source/upgrading_ruby_on_rails.md

```diff
@@ -365,6 +365,12 @@ versions, there are two scenarios to consider:
     config.active_record.encryption.hash_digest_class = OpenSSL::Digest::SHA1
     ```
 
+    If all of your data was encrypted non-deterministicly (the default unless `encrypts` is passed `deterministic: true`, you can instead configure SHA-256 for Active Record Encryption as in scenario 2 below and also allow columns previously encrypted with SHA-1 to be decrypted by setting:
```

Comment:
This is missing a `)` but it also repeats the paragraphs starting with "In addition, a new configuration".
```suggestion
    If all of your data was encrypted non-deterministicly (the default unless `encrypts` is passed `deterministic: true`), you can instead configure SHA-256 for Active Record Encryption as in scenario 2 below and also allow columns previously encrypted with SHA-1 to be decrypted by setting:
```